```
RestMLGrpcExecuteAgentStreamIT > testExecuteAgentStream_bedrockStreamsChunksEndToEnd FAILED
    java.lang.AssertionError: stream terminated with error: io.grpc.StatusRuntimeException: PERMISSION_DENIED: Error from remote service: User: arn:aws:sts::419213735998:assumed-role/github_ml_token_role/GitHubActions is not authorized to perform: bedrock:InvokeModelWithResponseStream on resource: arn:aws:bedrock:us-west-2:419213735998:inference-profile/global.anthropic.claude-sonnet-4-5-20250929-v1:0 because no identity-based policy allows the bedrock:InvokeModelWithResponseStream action (Service: BedrockRuntime, Status Code: 403, Request ID: fc6181b9-d29b-46d3-9bd4-295a388a1974) 
```

Fixed this by updating the policy of `github_ml_token_role` role to include `bedrock:InvokeModelWithResponseStream`

--- a/plugin/src/test/java/org/opensearch/ml/rest/RestMLRAGSearchProcessorIT.java
+++ b/plugin/src/test/java/org/opensearch/ml/rest/RestMLRAGSearchProcessorIT.java
@@ -207,6 +207,9 @@ public class RestMLRAGSearchProcessorIT extends MLCommonsRestTestCase {
         + "  \"description\": \"The connector to bedrock claude haiku 4.5 model\",\n"
         + "  \"version\": 1,\n"
         + "  \"protocol\": \"aws_sigv4\",\n"
+        + "  \"client_config\": {\n"
+        + "    \"max_connection\": 200\n"
+        + "  },\n"
         + "  \"parameters\": {\n"
         + "    \"region\": \""
         + GITHUB_CI_AWS_REGION

